src/model.py

```diff
@@ -675,6 +685,30 @@ def verify_loras(self, request):
             verified_request = request
         return verified_request
 
+    def _check_health(self, requests):
+        coro = self.llm_engine.check_health()
+        future = asyncio.run_coroutine_threadsafe(coro, self._loop)
+        try:
+            future.result()
+        except Exception as e:
+            self.logger.log_error(
+                f"[vllm] Engine is not healthy and model will be unloaded: {e}"
+            )
+            pb_utils.unload_model(self.model_config["name"])  # non-blocking
+            self._is_healthy = False
+        if not self._is_healthy:
+            for request in requests:
+                request.get_response_sender().send(
+                    pb_utils.InferenceResponse(
+                        error=pb_utils.TritonError(
+                            message="vLLM engine is not healthy and model will be unloaded",
```

Comment:
this goes to user, right? If so, I think user might be confused with loaded and unloaded terms

Reply:
Why the user is confused?

Reply:
First question if I'm unaware of Triton, what does that mean? Maybe something like: "... model is currently unavailable " ? What do you think?

Reply:
Ok. The user does not need to know if the model will be unloaded, which is an implementation detail. How about we change it to "Model is unavailable due to unhealthy vLLM engine"?

Reply:
[Update docs and messages](https://github.com/triton-inference-server/vllm_backend/pull/73/commits/b6bd6496627f7eb37ab94899a5eda98d3ffef48d)